source/GameInterface.Tests/Bootstrap/Patches/MapScenePatches.cs

```diff
@@ -0,0 +1,31 @@
+﻿using HarmonyLib;
+using SandBox;
+using System;
+using TaleWorlds.Core;
+using TaleWorlds.Library;
+
+namespace GameInterface.Tests.Bootstrap.Patches;
+
+[HarmonyPatch(typeof(MapScene))]
+internal class MapScenePatches
+{
+    [HarmonyPatch(nameof(MapScene.GetFaceTerrainType))]
+    [HarmonyPrefix]
+    static bool GetFaceTerrainTypePrefix(ref TerrainType __result)
+    {
+        Array values = Enum.GetValues(typeof(TerrainType));
+        Random random = new Random();
```

Comment:
Is there a reason to use Random here? Wouldn't this create different terrain types for every player?

Reply:
This is for testing only, these patches are located in GameInterface.Tests

Reply:
Haha oppps, missed that.